src/src/rx-serial/SerialMavlink.cpp

```diff
@@ -50,13 +51,13 @@ void SerialMavlink::sendQueuedData(uint32_t maxBytesToSend)
 SerialMavlink::SerialMavlink(Stream &out, Stream &in):
     SerialIO(&out, &in),
     // 255 is typically used by the GCS, for RC override to work in ArduPilot `SYSID_MYGCS` must be set to this value (255 is the default)
```

Comment:
The existing comments here need updating now that you have made these configurable (rather than force the defaults).